### DISCUSSION TITLE
job.<job id>.needs not working as documented

### DISCUSSION THREAD

**Discussion by mark-rushakoff** [community/community#26492] [Actions]
<p>I have a job like:</p>
<pre><code>lint:
    runs-on: ubuntu-latest
    needs: [lint_foo, lint_bar, lint_baz]
    steps:
    - run: true
</code></pre>
<p>I was hoping to use the “meta” job <code>lint</code> as a single status for other automation to confirm whether all the individual lint jobs succeeded.</p>
<p>However, I am seeing that if a single lint_foo or lint_bar or lint_baz step fails, the lint job doesn’t report at all. The documentation led me to believe that lint would report as failure.</p>
<p>According to <a href="https://help.github.com/en/articles/workflow-syntax-for-github-actions#jobsjob_idneeds" rel="nofollow noopener">https://help.github.com/en/articles/workflow-syntax-for-github-actions#jobsjob_idneeds</a>:</p>
<p>“If a job fails, all jobs that need it will also fail unless the jobs use a conditional statement that causes the job to continue.”</p>

**Comment by chrispat** [comment, marked as answer]
<p>That documentation needs to be clarified.   If you have a dependency on a previous job and that job fails by default your job will be skipped or fail to run at all.   Thank you for letting us know.</p>

**Comment by jannnik** [comment]
<p><a class="mention" href="/chrispat">@chrispat</a>, this issue does not seem to be solved since the documentation has not been carified. I have the same request for a ‘meta’ job and I am very confused on how to solve this problem!</p>
<p><a class="mention" href="/mark-rushakoff">@mark-rushakoff</a>  how did you solve this problem? </p>
<p>Edit: I solved it after I found this conversation: <a href="https://github.community/t/semantics-of-using-job-needs-and-job-if-together/17557/2">https://github.community/t5/GitHub-Actions/Semantics-of-using-quot-job-needs-quot-and-quot-job-if-quot/m-p/45495#M6231</a></p>
